polaris-patterns/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Copied from https://web.docs.shopify.io/components/Forms/DatePicker. This is just a wrapper component that supplies the Polaris date picker with `weekStartsOn={weekStartsOn}` using i18n. Should this just be added directly onto the existing polaris date picker?